Update landing page (#165)
ConsenSys/doc.tessera: docs/index.md

Ethereum clients such as [GoQuorum](https://docs.goquorum.consensys.net) and
[Hyperledger Besu](https://besu.hyperledger.org/en/stable/).

## What can you do with Tessera?

Tessera:

* Generates and maintains private/public key pairs.

* Self-manages and discovers all nodes in the network.

* Provides an API for communicating between Tessera nodes and an API for communicating with
  privacy-enabled Ethereum clients.

> * Provides two-way SSL using TLS certificates and various trust models like trust on first use (TOFU),
>   whitelist, and certificate authority.
> 
> * Supports IP whitelist.

Reviewer: ```suggestion
* Provides two-way SSL using TLS certificates and various trust models such as trust on first use (TOFU),
  allowlist, and certificate authority.

* Supports IP allowlist.
```

* Generates and maintains private/public key pairs.

* [Self-manages](Concepts/Privacy-Manager/Privacy-manager.md) and [discovers](Concepts/p2p-discovery.md) all nodes in the network.

> * Provides [an API](Reference/TesseraAPI.md) for communicating between Tessera nodes and an API for communicating with

Reviewer: ```suggestion
* Provides [an API](Reference/TesseraAPI.md) for communicating between Tessera nodes and with
```
